src/MenuButton.js

```diff
@@ -162,33 +181,26 @@ export default class MenuButton extends React.Component<Props, State> {
         ref={this._floatAnchorRef}
         options={positionOptions}
         zIndex={menuZIndex}
-        anchor={anchorRef =>
-          <ButtonComponent
-            domRef={el => {
-              this._anchorEl = el;
-              setRef(anchorRef, el);
-            }}
-            type="button"
-            className={className}
-            style={style}
-            onMouseDown={e => {
-              if (e.button !== 0) return;
-              this.toggle();
-            }}
-            onKeyPress={e=>{
-              if (e.key === 'Enter' || e.key === ' ') {
-                this.toggle();
-              }
-            }}
-            aria-haspopup={true}
-            aria-expanded={opened}
-            disabled={disabled}
-            title={title}
-            {...this.props.buttonProps}
-          >
-            {children}
-          </ButtonComponent>
-        }
+        anchor={anchorRef => {
+          if (!renderButton) {
+            return (
+              <button
+                className={className}
+                style={style}
+                ref={el => this._setRef(el, anchorRef)}
+                aria-haspopup={true}
+                aria-expanded={opened}
+                onKeyPress={e => this._onKeyPress(e)}
+                onMouseDown={e => this._onMouseDown(e)}
+                disabled={disabled}
+                title={title}
+              >
+                {this.props.children}
+              </button>
+            );
+          }
```

Comment:
Let's remove this if-block by assigning a default value to the `renderButton` variable, either through default props or `const renderButton = this.props.renderButton || ...;`